env/env.common.config.js

```diff
@@ -15,7 +15,9 @@ module.exports = {
         host: 'localhost',
         port: 9000
     },
-    public: {},
+    public: {
+        localStorageTokenKey: 'sh-token'
```

Comment:
I would put here only environment dependent vaiables, 
```localStorageTokenKey``` can be hardocded inside code